src/httpserver.cpp

```diff
@@ -385,13 +385,7 @@ bool InitHTTPServer()
     // Redirect libevent's logging to our own log
     event_set_log_callback(&libevent_log_cb);
 #if LIBEVENT_VERSION_NUMBER >= 0x02010100
-    // If -debug=libevent, set full libevent debugging.
-    // Otherwise, disable all libevent debugging.
-    if (LogAcceptCategory(BCLog::LIBEVENT)) {
-        event_enable_debug_logging(EVENT_DBG_ALL);
-    } else {
-        event_enable_debug_logging(EVENT_DBG_NONE);
-    }
+    event_enable_debug_logging(EVENT_DBG_ALL);
```

Comment:
Don't do this! This existed for a reason. What you did will reduce libevent performance extremely by always enabling their internal debugging.